packages/devtools_server/lib/src/server.dart

```diff
@@ -88,13 +92,18 @@ void serveDevTools({
     defaultDocument: 'index.html',
   );
 
+  final sseHandler = SseHandler(Uri.parse('/api/sse'))
+    ..connections.rest.listen(clients.acceptClient);
+
   // Make a handler that delegates to the correct handler based on path.
   final handler = (shelf.Request request) {
     return request.url.path.startsWith('packages/')
         // request.change here will strip the `packages` prefix from the path
         // so it's relative to packHandler's root.
         ? packHandler(request.change(path: 'packages'))
-        : buildHandler(request);
+        : request.url.path.startsWith('api/sse')
```

Comment:
I would move this logic into a few if statements each of which ends in a return. It is now too complex to be using conditionals like this).

Reply:
👍